### ISSUE TITLE
Flake check warnings

### ISSUE THREAD

**Issue by zimbatm** [hercules-ci/flake-parts#21]
I don't specify any darwinModules or overlays, but still get these errors:
```
warning: unknown flake output 'darwinModules'
warning: flake output attribute 'overlay' is deprecated; use 'overlays.default' instead
```
What's funny is that the `devShell` output is not recognized, so some of flake-parts has already been ported to the `.default` of Nix 2.8

**Comment by terlar** [comment]
I agree that it would be nice if `darwinModules` only was configured if there are any. Especially since it throws a warning.

**Comment by roberth** [comment]
`darwinModules` solved locally by
 - #80

Other attrs solved upstream in Nix 2.13 by
 - https://github.com/NixOS/nix/pull/7606
